src/envoy/http/authn/http_filter_factory.cc

```diff
@@ -42,7 +42,7 @@ class AuthnFilterConfig : public NamedHttpFilterConfigFactory,
     if (status.ok()) {
       return createFilter(policy_);
     } else {
-      PANIC("Utils::ParseJsonMessage() return value is NOT ok!");
+      PANIC("Utils::ParseJsonMessage() return value is: " + status.ToString());
       throw EnvoyException(
```

Comment:
Miss this in the last review :). I don't think we need both panic and throw exception. Panic already do it, I think, though I'm not 100% sure. If you want to be fully certain, LOG error/critical and throw exception.

Reply:
Based on the following defintion, PANIC is doing log only. So we should be OK.
 ENVOY_LOG_TO_LOGGER(Envoy::Logger::Registry::getLog(Envoy::Logger::Id::assert), critical,        \
                      "panic: {}", X);

Reply:
I take it back. PANIC also does abort().

Reply:
#define PANIC(X)                                                                                   \
  ENVOY_LOG_TO_LOGGER(Envoy::Logger::Registry::getLog(Envoy::Logger::Id::assert), critical,        \
                      "panic: {}", X);                                                             \
  abort();

Reply:
I have revised it to LOG critical and throw exception.